docs/plugins.md

```diff
@@ -13,7 +13,7 @@ complete control.
 - All plugins live in their own folders in ["src/plugins"](../src/plugins).
 - Plugin names should be in follow the format: `/[a-z_]+$`
 - JS source should live in a "plugin.js" file (required).
-- CSS should live in a "plugin.less" file (optional). It will be bundled at build time.
+- CSS should live in a "plugin.scss" file (optional) and include it into `src/scss/selectize.scss`. It will be bundled at build time.
```

Comment:
For compatibility with the older versions of bootstrap, there should be both a sass and less version of styles.

This might change in the future if we decide to drop support for older versions, but there is enough of a user base that still does use older versions that we should try to support them where possible

Reply:
sure, misstake, I thought it had not been updated, I had trouble finding what I was missing to use my SCSS file